observationtree/src/main/scala/explore/observationtree/ObsBadge.scala

```diff
@@ -1,23 +1,30 @@
 // Copyright (c) 2016-2021 Association of Universities for Research in Astronomy, Inc. (AURA)
 // For license information see LICENSE or https://opensource.org/licenses/BSD-3-Clause
 
-package explore.components
+package explore.observationtree
 
 import cats.syntax.all._
+import explore.Icons
 import explore.components.ui.ExploreStyles
 import explore.model.ObsSummary
 import explore.model.reusability._
 import japgolly.scalajs.react._
 import japgolly.scalajs.react.vdom.html_<^._
+import lucuma.core.model.Observation
 import lucuma.core.util.Enumerated
 import react.common._
+import react.common.implicits._
+import react.semanticui.elements.button.Button
+import react.semanticui.sizes._
 import react.semanticui.views.card._
 
 final case class ObsBadge(
   obs:      ObsSummary,
   layout:   ObsBadge.Layout,
-  selected: Boolean = false
+  selected: Boolean = false,
+  deleteCB: Observation.Id => Callback
```

Comment:
Would it be OK to expose `IO` instead of `Callback`? Just to use the same style throughout.